File: lib/models/member.dart
enum MemberStatus {
  active,
  suspended,
  dismissed,
}

class Member {
  final String id;
  final String name;
  final String location;
  final bool isPaid;
  final MemberStatus status;

  const Member({
    required this.id,
    required this.name,
    required this.location,
    this.isPaid = false,
    this.status = MemberStatus.active,
  });

  Member copyWith({
    String? id,
    String? name,
    String? location,
    bool? isPaid,
    MemberStatus? status,
  }) {
    return Member(
      id: id ?? this.id,
      name: name ?? this.name,
      location: location ?? this.location,
      isPaid: isPaid ?? this.isPaid,
      status: status ?? this.status,
    );
  }
}

File: springboot-backend/src/main/java/com/veteranapp/backend/model/Member.java
package com.veteranapp.backend.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Member {
    private String id;
    private String organizationId;
    private String name;
    private String location;
    private Boolean isPaid;
    private String status;
    private String role;
    private String service;
}
